### ISSUE TITLE
CRA example

### ISSUE THREAD

**Issue by phmatray** [Igorbek/typescript-plugin-styled-components#38]
Would it be possible to add an example to the README showing how to configure typescript-plugin-styled-components with CRA2 --typescript?

**Comment by Igorbek** [comment]
Thank you for filing a request.

CRA2 uses Babel to transpile code and does not use any webpack typescript loaders.
First of all, there's another solution for Babel: [`babel-plugin-styled-components`](https://github.com/styled-components/babel-plugin-styled-components). It might be sufficient for your goals.
Second, after some research it seems to be impossible to configure Babel's typescript transformer to use custom TypeScript transformers which the current plugin is.

I'll appreciate any information or examples from people who used this plugin with babel/CRA.
And I will continue digging into that, but it's not a priority until any useful insights come up.

**Comment by atatakobry** [comment]
FYI: CRA allows to start project with typescript support, so this plugin looks very interesting. Unfortunately, I have no idea how to make it works :(

**Comment by tuzmusic** [comment]
> Thank you for filing a request.
> 
> CRA2 uses Babel to transpile code and does not use any webpack typescript loaders. First of all, there's another solution for Babel: [`babel-plugin-styled-components`](https://github.com/styled-components/babel-plugin-styled-components).

Are you saying that your the babel plugin will work for CRA with TS, and that your plugin is only necessary for custom TS installations?

**Comment by Igorbek** [comment]
That's right, this plug-in is not for Babel. It is for such setups where the TypeScript compiler is used.

**Comment by tuzmusic** [comment]
@atatakobry @phmatray I've just succeeded in getting the "original" babel plugin to work in my CRA typescript app by following the instructions [here](https://dev.to/ansonh/simplest-way-to-install-babel-plugins-in-create-react-app-7i5).
